### ISSUE TITLE
New citation mode: possessive cite or so

### ISSUE THREAD

**Issue by denismaier** [andras-simonyi/citeproc-el#26]
When using in-text citation you sometimes may want to have the author of a paper in the possessive form:

> I'm talking about Doe's (2020) concept of ...

Should this be supported?
Related pandoc issue: https://github.com/jgm/pandoc/issues/7385

Should be much easier to support this with the new org syntax than with pandoc's design philosophy in mind.

**Comment by andras-simonyi** [comment]
Isn't this too English-dependent/focused? I don't really see how this would work in a language-agnostic way -- in morphologically rich(er) languages you could have several additional cases like dative, accusative etc.

**Comment by denismaier** [comment]
> Isn't this too English-dependent/focused? I don't really see how this would work in a language-agnostic way -- in morphologically rich(er) languages you could have several additional cases like dative, accusative etc.

Yes, that's probably correct. I don't have a good suggestion ATM.
